corehq/apps/README.rst

```diff
@@ -0,0 +1,190 @@
+registration
+   Workflows for creating new accounts on HQ.
+reminders
+   A subset of SMS/messaging, including functionality around incoming SMS keywords. "Reminders" is a leftover term from a previous iteration of the messaging framework.
+saved_reports
+   Functionality to let users save a specific set of report filters and optionally run reports with those filters on a scheduled basis.
+toggle_ui
+   Framework for feature flags, which are used to limit internatl feature to specific domains and/or users.
+translations
+   Functionality for managing application translations, including integration with Transifex, which is used by a small number of projects.
+user_importer
+   Bulk importing of users.
+
+Tertiary Apps
+^^^^^^^^^^^^^
+These apps may be useful parts of the system but don't have as much active development as the groups above.
+
+aggregate_ucrs
+   TODO
+analytics
+   Integrations with third-party analytics tools such as Google Analytics and Kissmetrics.
+   Also contains internal product-focused tools such as AB testing functionality.
+builds
+   Models relating to CommCare Mobile builds, so that app builders can control which mobile version their apps use.
+case_search
+   TODO
+dashboard
+   The tiled UI that acts as the main landing page for HQ.
+formplayer_api
+   Functionality interacting with formplayer, primarily used by SMS surveys.
+mobile_auth
+   Generates the XML needed to authorize mobile users.
+notifications
+   "Banner" notifications used by the support team to notify users of upcoming downtime,
+   ongoing issues, etc.
+ota
+   Functionality at the interface of CommCare HQ and CommCare Mobile: demo users, device logs, mobile recovery, etc.
+receiverwrapper
+   TODO
+settings
+   API keys and 2FA functionality.
+smsbillables
+   Billing functionality relating to charging for SMS, allowing us to pass carrier charges on to clients.
+smsforms
+   SMS surveys, a part of messaging that allow end users to interact with a CommCare form via SMS instead of
+   via mobile or Web Apps.
+styleguide
+   Documentation of best practices for UI development, including live examples of common patterns.
+zapier
+   Integration with `Formplayer <https://zapier.com/>`_
+
+Engineering Apps
+^^^^^^^^^^^^^^^^
+These apps are developer-facing tools.
+
+cachehq
+   Caching functinality for CouchDB.
+case_migrations
+   Functionality to support users defining and excuting data migrations on cases. Candidate for deprecation.
+change_feed
+   Infrastructure for propagating changes in primary data stores (couch, postgres) to secondary sources (ElasticSearch).
+cleanup
+   Miscellaneous commands for cleaning up data: deleting duplicate mobile users, deleting couch documents for models that have been moved to postgres, etc.
+couch_sql_migration
+   Utiltiy code for migration form and case data from couch to postgres.
+data_analytics
+   Internal impact-related metrics.
+data_pipeline_audit
+   TODO
+domain_migration_flags
+   TODO
```

Comment:
@snopoke or @kaapstorm Can you replace this with a sentence?

Reply:
```suggestion
   Dynamic flags that are used to indicate when a data migration is taking place for a specific domain. The flags are
   checked in various places thought the code and will restrict access to certain features when enabled. These flags
   are set during large data migrations such as moving case & form data from Couch -> SQL, migrating a domain to a
   different CommCare instance.
```